bin/cli.js

```diff
@@ -114,11 +116,11 @@ async function generateSchema(pathToSpec) {
 
   // output
   if (output === OUTPUT_FILE) {
-    let outputFilePath = path.resolve(process.cwd(), flags.output); // note: may be directory
```

Comment:
Overdue cleanup: `URL()` is a vastly-preferred way to work with the filesystem in Node, because it’s not only faster, but it also works in both Windows and *nix systems without any fuss. And no more awkward `path.join()` or conditional `path.sep` logic; simply use forward slashes and It Just Works™

`path.resolve()` is a bit of a convoluted mess under-the-hood. It’s great when it works, but it’s a headache when it doesn’t.